.changeset/renovate-0e8494e.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/repo-tools': patch
```

Comment:
Is this a breaking change if people now need to create a `.prettierrc.yaml`?

Reply:
The idea was that this should still keep working with prettier 2 if that's what you're depending on in your repo, because that resolve call should just return nothing if the file doesn't exist. Maybe worth triple checking ... Gimme a sec to verify again

Reply:
OK let's go back to draft until i have time to look at this again